recipes/lda/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "lda" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 50b91f80aff1b138b8b78f48d581c0e685e7adb393da9e45905773d5f4cc8bf5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py>=37]
```

Comment:
If building without skipping `py>=37` then an error like this occurs: https://github.com/kevlar-dev/kevlar/issues/282

If there is a way to fix and allow building on py>=37, please let me know

Reply:
don't you need ``cython`` for this project?